tests/vcs/git/GitCommitBearTest.py

```diff
@@ -109,17 +109,17 @@ def test_pure_oneliner_message(self, patch):
         self.assertTrue(self.msg_queue.empty())
 
     def test_shortlog_checks_length(self):
-        self.git_commit("Commits messages that nearly exceeds default limit")
+        self.git_commit("Commit messages that nearly exceed default limit")
 
         self.assertEqual(self.run_uut(), [])
         self.assertTrue(self.msg_queue.empty())
 
         self.assertEqual(self.run_uut(shortlog_length=17),
-                         ["Shortlog of HEAD commit is 33 character(s) "
-                          "longer than the limit (50 > 17)."])
+                         ["Shortlog of HEAD commit is 31 character(s) "
+                          "longer than the limit (48 > 17)."])
         self.assertTrue(self.msg_queue.empty())
 
-        self.git_commit("A shortlog that is too long is not good for history")
+        self.git_commit("Very long shortlog never good for a project history")
```

Comment:
Separate commit? It's a part of test. If I make these changes in separate commit it'd possibly fail?